site/content/docs/Advanced/_index.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "Advanced Topics"
 linkTitle: "Advanced Topics"
-weight: 99
+weight: 35
```

Comment:
What's the proposed new order?

IOW, what is the order of the pages before and after this change?

Reply:
The original order was:

- Features
- Supported Storage Locations
- Getting Started Guide
- Installation Guide
- FAQ
- Command Line Reference
- Release Notes
- Upgrading to a new version
- Advanced Topics

The new order is:
- Features
- Getting Started Guide
- Installation Guide
- Supported Storage Locations
- Repository Server
- Advanced Topics
- Command Line Reference
- Upgrading to a new version
- Release Notes
- FAQ